docs/src/modules/java-protobuf/pages/workflows.adoc

```diff
@@ -179,6 +179,36 @@ IMPORTANT: For simplicity purposes, we are returning the internal state directly
 
 A full transfer workflow source code is available https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples/java-protobuf-transfer-workflow[here (Java Protobuf), {tab-icon}, window="new"] or https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples/scala-protobuf-transfer-workflow[here (Scala Protobuf), {tab-icon}, window="new"]. Follow the `README` file to run and test it.
 
+
+== Deleting state
+
+If you want to delete the workflow state, you can use the `delete` method from the effect API. This will remove the workflow from the system.
```

Comment:
Seems more clear when we say that we are deleting the workflow rather than deleting the workflow state (as if something would still remain).

```suggestion
If you want to delete the workflow, you can use the `delete` method from the effect API.
```